tf/tfprocess.py

```diff
@@ -104,6 +117,13 @@ def __init__(self, cfg):
         self.RESIDUAL_BLOCKS = self.cfg['model']['residual_blocks']
         self.SE_ratio = self.cfg['model']['se_ratio']
         self.policy_channels = self.cfg['model'].get('policy_channels', 32)
+        self.policy_embedding_size = self.cfg['model'].get('policy_embedding_size', self.RESIDUAL_FILTERS)
+        self.encoder_layers = self.cfg['model'].get('encoder_layers', 1)
+        self.encoder_heads = self.cfg['model'].get('encoder_heads', 4)
```

Comment:
default of 4 heads seems an interesting choice - if filters is 64 that would mean only 16 per head in the default.